### ISSUE TITLE
Update integration with HTML

### ISSUE THREAD

**Issue by fred-wang** [WICG/private-network-access#56]
From https://wicg.github.io/private-network-access/#integration-html:

>  The initialize the Document object algorithm is amended with an extra step in between the existing steps 10 and 11, before the new Document's CSP list is initialized:
>  12  Set document’s IP address space to response’s IP address space.

new Document's CSP list is step 12 of https://html.spec.whatwg.org/multipage/browsing-the-web.html#initialise-the-document-object ; so I think this should be " existing steps 11 and 12".

> The run a worker algorithm is amended with two extra steps in between the existing steps 14.4 and 14.5, after the new WorkerGlobalScope's referrer policy is set:

In current version of https://html.spec.whatwg.org/multipage/workers.html#run-a-worker, this happens in 13.6, so I guess it's "between 13.6 and  13.7"?

**Comment by letitz** [comment]
Thanks for the corrections!

**Comment by annevk** [comment]
I suspect this should be integrated with policy container?

**Comment by letitz** [comment]
Yes! The `address space` property of documents is implemented using PolicyContainer in Chrome, and that should be reflected in the spec.

**Comment by letitz** [comment]
Wow, am I glad that the policy container exists. That was so easy! Please feel free to re-open if you notice other errors.
